BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -376,6 +391,21 @@ public override void OnStart(StartState state)
         {
             //base.OnStart(state);
 
+            initialMass = part.mass;
+            if (boosterMass < 0 || boostTime <= 0)
+            {
+                if (BDArmorySettings.DEBUG_MISSILES && boosterMass < 0) Debug.LogWarning("[BDArmory.MissileBase]:Error Bad Configured Missile");
+                boosterMass = 0;
+            }
+            else if (useFuel && boostTime > 0 && boosterMass <=0) boosterMass = DefaultMass();
+            
+            
+            if (sustainerMass < 0 || cruiseTime <= 0)
+            {
+                if (BDArmorySettings.DEBUG_MISSILES && sustainerMass < 0) Debug.LogWarning("[BDArmory.MissileBase]:Error Bad Configured Missile");
+                sustainerMass = 0;
```

Comment:
Same here, except it'd be ``if (sustainerMass >(initialMass - boosterMass)*0.8f) { sustainerMass = (initialMass - boosterMass)*0.8f}`` or ``sustainerMass = MathF.Clamp(sustainerMass , initialMass*0.1f, (initialMass - boosterMass)*0.8f);`` check.